www/main.js

```diff
@@ -215,6 +237,12 @@ function createEnablerList() {
 
   enablerList.appendChild(checkButtonBar);
   
+  header.appendChild(headerPrefix);
+  header.appendChild(enabledCount);
+  header.appendChild(totalCount);
+  header.appendChild(headerSuffix);
```

Comment:
This seems needlessly complex.  Why not just a single text element, and we set the whole "Running X of Y" text each time?  We already append the '/' to the first span and we aren't using any fancy data binding.